snap/pack/pack_test.go

```diff
@@ -36,6 +37,9 @@ import (
 	"github.com/snapcore/snapd/snap/pack"
 	"github.com/snapcore/snapd/snap/squashfs"
 	"github.com/snapcore/snapd/testutil"
+
+	// for SanitizePlugsSlots
+	_ "github.com/snapcore/snapd/interfaces/builtin"
```

Comment:
do we need this now?

Reply:
Yes, without that the SanitizePlugSlots is not set and we get:
... Panic: SanitizePlugsSlots function not set (PC=0x4588BA)

for every test.